OptimizelySDK.Tests/OdpSegmentsDatafile.json

```diff
@@ -0,0 +1,318 @@
+{
+  "groups": [
+  ],
+  "environmentKey": "production",
+  "rollouts": [
+    {
+      "experiments": [
+        {
+          "status": "Running",
+          "audienceConditions": [
+          ],
+          "audienceIds": [
+          ],
+          "variations": [
+            {
+              "variables": [
+              ],
+              "id": "109682",
+              "key": "off",
+              "featureEnabled": false
+            }
+          ],
+          "forcedVariations": {
+          },
+          "key": "default-rollout-36645-22702840437",
+          "layerId": "rollout-36645-22702840437",
+          "trafficAllocation": [
+            {
+              "entityId": "109682",
+              "endOfRange": 10000
+            }
+          ],
+          "id": "default-rollout-36645-22702840437"
+        }
+      ],
+      "id": "rollout-36645-22702840437"
+    }
+  ],
+  "typedAudiences": [
+    {
+      "id": "22727912729",
+      "conditions": [
+        "and",
+        [
+          "or",
+          [
+            "or",
+            {
+              "value": "ats_bug_bash_segment_gender",
```

Comment:
Use the same segment name in other audiences, so we can validate they're not repeated in the generated "Segments" list.

Reply:
Yup. `ats_bug_bash_segment_gender` is now repeated in the `Segments` array. I fixed my unit test to look for this scenario, and it's now failing. 👍 

I guess DatafileProjectConfig should dedupe into a `SortedSet` before assigning to `string[]`